src/components/ProjectDetailsComponent.js

```diff
@@ -26,55 +44,106 @@ const ProjectDetailsComponent = () => {
+        setState({
+          snackbarOpen: true,
+          snackbarMessage: 'Name already in use, could not update name'
+        });
+        return;
+      }
+      await db.updateProject(state);
+      setState({
+        snackbarOpen: true,
+        snackbarMessage: 'Successfully updated project information!'
+      });
+    } catch (err) {
+      setState({
+        snackbarOpen: true,
+        snackbarMessage: `Project update failed: ${err.message}`
+      });
+      console.log(`Project update failed: ${err}`);
     }
+  };
 
-    console.log('sending state: ', state);
-    request = await fetch(`http://localhost:5000/api/addProject`, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(state)
-    });
-    results = await request.json();
-    console.log(results);
+  const handleSnackbarClose = (event, reason) => {
+    if (reason === 'clickaway') return;
+    setState({ snackbarOpen: false, snackbarMessage: '' });
   };
 
   return (
     <div>
       <h1 style={styles.pageTitle}>Project Details</h1>
       <div style={styles.form}>
-        <label className='inputPrompt'>
-          Project Name<span style={{ color: 'red' }}>*</span>:
-        </label>
-        <input type='text' onChange={handleProjectNameChange} />
+        <TextField
+          className={classes.inputContainer}
+          defaultValue={state.projectName}
+          label='Project Name'
+          required={true}
+          InputLabelProps={{ className: classes.inputLabel }}
+          InputProps={{ className: classes.textField }}
+          onChange={handleProjectNameChange}
+        />
         <br />
-        <label className='inputPrompt'>Company Name:</label>
-        <input type='text' onChange={handleCompanyNameChange} />
+        <TextField
+          className={classes.inputContainer}
+          label='Company Name'
+          onChange={handleCompanyNameChange}
+          InputProps={{ className: classes.textField }}
+          InputLabelProps={{ className: classes.inputLabel }}
+        />
         <br />
-        <label className='inputPrompt'>Project Desciption:</label>
-        <input type='text' onChange={handleDescriptionChange} />
+        <TextField
+          className={classes.inputContainer}
+          label='Description'
+          onChange={handleDescriptionChange}
+          InputProps={{ className: classes.textField }}
+          InputLabelProps={{ className: classes.inputLabel }}
+        />
         <br />
-        <input
-          type='button'
-          onClick={handleSubmitButton}
-          value='Submit'
-          disabled={!state.projectName}
+        <Button
+          variant='contained'
+          color='primary'
+          onClick={project ? handleUpdateButton : handleSubmitButton}
+          disabled={state?.projectName === ''}>
+          {project ? 'Update' : 'Submit'}
+        </Button>
+        <Snackbar
```

Comment:
I know you mentioned having this in the App.js so we might want to do that sooner rather than later.